frontend/style.css

```diff
@@ -16,132 +18,211 @@ html, body {
 }
 
-#menu > :nth-child(2) {
-  background-color: #028090;
+.option.selected {
+  font-weight: bold;
 }
 
-#menu > :nth-child(3) {
-  background-color: #00a896;
+.option.selected:after {
+  content: ' ◄';
+  display: inline;
 }
 
-#menu > :nth-child(4) {
-  background-color: #02c39a;
+/* menu button colors */
+#menu > :nth-child(1) { background-color: #75c9c8; }
+#menu > :nth-child(2) { background-color: #9CB6DD; }
+#menu > :nth-child(3) { background-color: #c0b9dd; }
+#menu > :nth-child(4) { background-color: #ded9e2; }
+#menu > :nth-child(5) { background-color: #f7f4ea; }
+
+#settings {
+  min-width: 25em;
 }
 
-#menu > :nth-child(5) {
-  background-color: #f0f3bd;
+.slider-container {
+  display: block;
+  margin-top: .5em;
+  margin-bottom: .5em;
 }
 
-#slider-container {
-  text-align: center;
+.slider-container:last-child {
+  margin-bottom: 0;
 }
 
-#slider-container > input {
-  margin-bottom: 20px;
+.slider-container label .label {
+  display: inline-block;
+  margin-right: 1em;
+  text-transform: capitalize;
+  min-width: 10ch;
 }
 
-.slider-label {
-  margin-bottom: 5px;
+.slider-container input {
+  cursor: pointer;
 }
 
-#delay {
-  display: flex;
-  flex-direction: row;
-  justify-content: space-evenly;
+.slider-container .value {
+  display: inline-block;
+  min-width: 6ch;
+  text-align: right;
+  margin-left: 1ch;
+  margin-right: 1ch;
 }
 
-/* Cursor */
-.menu-item {
+.slider-container .reset {
+  /* icon, by default too big */
+  font-size: 80%;
   cursor: pointer;
 }
 
-.window {
+.slider-container .default .reset {
+  opacity: 0.5;
   cursor: default;
-  pointer-events: none;
 }
 
-#env > div:nth-child(n+3):nth-last-child(n) {
-  cursor: pointer;
-  pointer-events: all;
+.toggle-container {
+  display: block;
+  margin-top: 0.5em;
+  margin-bottom: 0.5em;
+}
+
+.toggle-container:last-child {
+  margin-bottom: 0;
+}
+
+.toggle-container label .label {
+  display: inline-block;
+  margin-right: 1em;
+  text-transform: capitalize;
+  min-width: 10ch;
+}
+
+.explanation {
+  padding-left: 1em;
+  font-style: italic;
 }
 
-#slider-container > input {
+
+#delay {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-evenly;
+}
+
+#env > div {
   cursor: pointer;
-  pointer-events: all;
 }
 
 #delay > div {
   cursor: pointer;
-  pointer-events: all;
 }
 
 #ambient, #overshoot, #wireframe {
   cursor: pointer;
-  pointer-events: all;
 }
 
-/* Hidden */
-.hidden {
-  display: none;
+/* Axes view */
+.axes-frame {
+  position: absolute;
+  bottom: 15px;
+  left: 15px;
+
+  height: 8em;
+  width: 8em;
+
+  /* labels */
+  font-family: 'Arial Black', sans-serif;
+  font-weight: bold;
 }
 
-/* Underline */
-.selected{
-  text-decoration: solid underline;
+/* let clicks pass through the axes view */
+.axes-frame,
+.axes-frame * {
+  pointer-events: none;
 }
 
-/* Underline */
-.disabled{
-  text-decoration: solid line-through;
+#controls {
+  box-sizing: border-box;
+  width: auto;
+  width: calc(100vw - 4em - 32px);
+
```

Comment:
- Is not the `width: auto` pointless? What is it purpose if it's redefined right after?
- Maybe it's because of the previous point, but the controls panel take all the width of my screen, and it looks very non-esthetics

Reply:
and yes, it is too wide, I haven't yet found a better responsive solution

Reply:
to clarify why it's pointless: the `width:auto` is there in case `calc()` isn't supported; but of course `auto` is the default value anyway…